alphabet.py

```diff
@@ -0,0 +1,29 @@
+from graph import Graph
+
+
+class Alphabet:
+    def __init__(self):
+        self.letters = set()
```

Comment:
Could you add a comment on what this does? I assume that you are creating an empty set for the letters, right?